src/maxminddb.c

```diff
@@ -281,6 +281,10 @@ LOCAL int map_file(MMDB_s *const mmdb)
         goto cleanup;
     }
     mmh = CreateFileMappingA(fd, NULL, PAGE_READONLY, 0, size, NULL);
+    if (mmh == NULL) {  /* Microsoft documentation for CreateFileMapping indicates this returns NULL not INVALID_HANDLE_VALUE on error */
```

Comment:
We try to write this as `NULL == mmh` to avoid accidentally typing `=` where we mean `==`.